document_quick_access/views/document_quick_access_rule.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 Creu Blanca
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record model="ir.ui.view" id="document_quick_access_rule_form_view">
+        <field
+            name="name"
+        >document.quick.access.rule.form (in document_quick_access)</field>
+        <field name="model">document.quick.access.rule</field>
+        <field name="arch" type="xml">
+            <form>
+                <header />
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button
```

Comment:
![imagen](https://user-images.githubusercontent.com/28590170/109619979-ac5c1400-7b39-11eb-866e-2f43c6e46e90.png)
Change it please on the new way of 14.0